tikv/raftstore/coprocessor.go

```diff
@@ -45,31 +57,51 @@ type splitCheckerHost struct {
 }
 
 func (spCheckerHost *splitCheckerHost) skip() bool {
-	//Todo, currently it is a place holder
-	return false
-}
-
-func (spCheckerHost *splitCheckerHost) onKv(observerCtx *observerContext, spCheKeyEntry splitCheckKeyEntry) bool {
-	// Todo: currently it is a place holder
+	return len(spCheckerHost.checkers) == 0
+}
+
+/// onKv is a hook called for every check during split.
+/// Return true means abort early.
+func (spCheckerHost *splitCheckerHost) onKv(region *metapb.Region, spCheKeyEntry *splitCheckKeyEntry) bool {
```

Comment:
use struct for `splitCheckKeyEntry` can avoid one memory allocation

Reply:
ok, thanks